test/spec/EditorCommandHandlers-test.js

```diff
@@ -1902,18 +2257,33 @@ define(function (require, exports, module) {
                 });
 
                 it("should handle multiple selections in different regions, toggling line selection (but falling back to block selection in HTML/CSS)", function () {
-                    myEditor.setSelections([{start: {line: 1, ch: 4}, end: {line: 1, ch: 10}},
+                    if (shouldIndentLineComment) {
```

Comment:
Should be separated to with / without `shouldIndentLineComments` like the other tests